examples/getting_started/README.md

```diff
@@ -0,0 +1,607 @@
+within the us-east-1 AWS region.
+
+testing.tfvars
+```
+region = "us-east-1"
+instance_type = "t2.medium"
+ami = "ami-6869aa05"
+```
+
+Now create your test kitchen instance
+```
+  $ bundle exec kitchen converge
+```
+
+Now run the tests
+```
+  $ bundle exec kitchen verify
+```
+
+Whoops!  Looks like we got an error this run!
+
+```
+>>>>>> ------Exception-------
+>>>>>> Class: Kitchen::ActionFailed
+>>>>>> Message: 1 actions failed.
+>>>>>>     Verify failed on instance <default-ubuntu>.  Please see
+.kitchen/logs/default-ubuntu.log for more details
+```
+
+So let's take a look at the .kitchen/log/default-ubuntu.log.  Parsing
+through the output, we see this:
+```
+  ERROR -- default-ubuntu: Message: Transport error, can't connect to
+  'ssh' backend: SSH session could not be established
+```
+
+This is because of the verifier in our .kitchen.yml
+
+.kitchen.yml
+```
+verifier:
+  name: terraform
+  format: doc
+  groups:
+    - name: default
+      tests:
+        - operating_system
+      hostnames: public_dns
+      username: ubuntu
+```
+
+Notice that the username we have specified for our test is "ubuntu" -
+this is fine for an ubuntu instance, but for an Amazon Linux instance,
+we need to use the "ec2-user" username
+
+So let's go ahead and change that, the verifier section of your
+.kitchen.yml should now look like this:
+
+.kitchen.yml
+```
+verifier:
+  name: terraform
+  format: doc
+  groups:
+    - name: default
+      tests:
+        - operating_system
+      hostnames: public_dns
+      username: ec2-user
+```
+
+Now try running the tests again
+
+```
+  $ bundle exec kitchen verify
+```
+
+And now we see a test failure
+
+```
+   1) Command lsb_release -a stdout should match /Ubuntu/
+      Failure/Error: DEFAULT_FAILURE_NOTIFIER =
+      lambda { |failure, _opts| raise failure }\
+      expected "" to match /Ubuntu/
+       Diff:
+       @@ -1,2 +1,2 @@
+       -/Ubuntu/
+       +""
+```
+
+Alright, that means our test is failing when it is supposed to be
+failing, and passing when it is supposed to pass.
+
+Open up your .kitchen.yml file and change the username back to ubuntu
+
+.kitchen.yml
+```
+verifier:
+  name: terraform
+  format: doc
+  groups:
+    - name: default
+      tests:
+        - operating_system
+      hostnames: public_dns
+      username: ubuntu
+```
+
+And open up your testing.tfvars file and switch back to an Ubuntu AMI
+
+testing.tfvars
+```
+region = "us-east-1"
+instance_type = "t2.medium"
+ami = "ami-fce3c696"
+```
+
+Then destroy your current kitchen instances
+
+```
+  $ bundle exec kitchen destroy
+```
+
+Then converge again
+
+```
+  $ bundle exec kitchen converge
+```
+
+And then run verify one more time, now you should see everything pass
+
+```
+  $ bundle exec kitchen verify
+```
+
+Huzzah!  This includes our "Getting Started" tutorial.
+
+Now, make sure to destroy your test instances
+
+```
+  $ bundle exec kitchen destroy
+```
```

Comment:
I got through all of it! Just had to change the instance size and add my IP to the default security group.

Something to think of for the future, I think if we had one less converge/verify session it would be a much more straight forward. Basically have a failing test in the very beginning, then passing test after fixing the code or such...